src/wp-includes/class-wp-theme.php

```diff
@@ -194,6 +194,25 @@ final class WP_Theme implements ArrayAccess {
 	 */
 	private $cache_hash;
 
+	/**
+	 * Block template folders.
+	 *
+	 * @since 6.4.0
+	 * @var array|null
+	 */
+	private $block_template_folders;
+
+	/**
+	 * Default values for template folders.
+	 *
+	 * @since 6.4.0
+	 * @var array
```

Comment:
```suggestion
	 * @var string[]
```

Similar to `private static $tag_map`